mmv1/products/filestore/api.yaml

```diff
@@ -225,6 +225,19 @@ objects:
                 A list of IPv4 or IPv6 addresses.
               output: true
               item_type: Api::Type::String
+            - !ruby/object:Api::Type::Enum
+              name: 'connectMode'
+              description: |
+                The network connect mode of the Filestore instance.
+                If not provided, the connect mode defaults to
+                DIRECT_PEERING.
+              input: true
+              min_version: beta
+              default_value: :DIRECT_PEERING
+              values:
+                - :CONNECT_MODE_UNSPECIFIED
```

Comment:
If you send the string `"CONNECT_MODE_UNSPECIFIED"` does the API send it back, or does it send a null? If it sends a null, we don't want this to be a configurable option.

Reply:
I'm assuming the only way to know is to test this and capture raw HTTP API responses?

Reply:
I tested this using the API explorer. Request Body:
```
{
  "networks": [
    {
      "connectMode": "CONNECT_MODE_UNSPECIFIED",
      "network": "default"
    }
  ],
  "fileShares": [
    {
      "name": "test",
      "capacityGb": 1024
    }
  ],
  "tier": "BASIC_HDD"
}
```
The response just showed done.
Response:
```
{
  "name": "projects/sam-playground-123/locations/us-central1-a/operations/operation-1630632919407-5cb0d4feb9b76-c6f6b048-9dde8dbb",
  "metadata": {
    "@type": "type.googleapis.com/google.cloud.common.OperationMetadata",
    "createTime": "2021-09-03T01:35:19.748196379Z",
    "target": "projects/sam-playground-123/locations/us-central1-a/instances/test1",
    "verb": "create",
    "cancelRequested": false,
    "apiVersion": "v1beta1"
  },
  "done": false
}
```

So it doesn't seem to be sending back null but it also doesn't send it back. Not sure what to do with this. I can try to see what the behaviour is for the other values.

Reply:
I probably misunderstood your original comment. The behaviour seems to indicate that CONNECT_MODE_UNSPECIFIED can be removed from the possible values since that would mean it's actually set to the default which is `DIRECT_PEERING`? I'm a bit confused by this behaviour honestly but this might be normal to you.

Reply:
Sorry, right, API explorer doesn't follow the operation so you have to do that by hand. Doing a Get afterwards was correct.

Your test showed that it's not a real value that can be specified, and the resource took on the default value- we can omit it as an option as setting it would just cause problems for users. These "X_UNSPECIFIED" values are an artifact of how GCP APIs are represented internally, so this is pretty standard.